lectures/cagan_ree.md

```diff
@@ -246,41 +243,33 @@ First, we store parameters in a `namedtuple`:
 ```{code-cell} ipython3
 # Create the rational expectation version of Cagan model in finite time
 CaganREE = namedtuple("CaganREE", 
-                        ["m0", "T", "μ_seq", "α", "δ", "π_end"])
+                        ["m0", "μ_seq", "α", "δ", "π_end"])
```

Comment:
Please stack these vertically and add a comment after each one.

Comments should not be capitalized unless they are a full sentence.  

Comments can be removed from `create_cagan_model` below -- it's best to put them where the symbol first appears.